src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/corporate/CorporateMappingResource.kt

```diff
@@ -167,13 +167,13 @@ class CorporateMappingResource(private val service: CorporateService) {
       ),
     ],
   )
-  suspend fun createMapping(
+  suspend fun createOrganisationMapping(
```

Comment:
there was a lot of overloading based on type - so a lot of function renames to compensate now we use the same type